frontend/src/component/feature/FeatureToggleList/FeatureToggleFilters/LifecycleFilters.tsx

```diff
@@ -47,22 +49,45 @@ const lifecycleOptions: {
     { label: 'Cleanup', value: 'completed' },
 ];
 
+const getStageCount = (
+    lifecycle: LifecycleStage['name'] | null,
+    lifecycleCount?: FeatureLifecycleCountSchema,
+) => {
+    if (!lifecycleCount) {
+        return undefined;
+    }
+
+    if (lifecycle === null) {
+        return (
+            (lifecycleCount.initial || 0) +
+            (lifecycleCount.preLive || 0) +
+            (lifecycleCount.live || 0) +
+            (lifecycleCount.completed || 0)
+        );
+    }
+
+    const key = lifecycle === 'pre-live' ? 'preLive' : lifecycle;
```

Comment:
wondering if we should avoid pre-live in frontend altogether and stick to preLive. Does our backend ever return pre-live?

Reply:
I think it does in some places